src/main/java/org/folio/linked/data/configuration/kafka/KafkaProducerConfiguration.java

```diff
@@ -50,6 +51,16 @@ public FolioMessageProducer<ResourceIndexEvent> authorityMessageProducer(
     return producer;
   }
 
+  @Bean
```

Comment:
Not sure why we need this bean.

I think there is a misunderstanding about the requirement for this ticket.

Purpose of this ticket is to update the "suppress -> FromDiscovery" and "suppress -> Staff" flags of clild instances in `linked-data-work` search index.

In order to do this, there is no need to define this new `instanceMessageProducer` bean. We should use the existing `bibliographicMessageProducer` bean instead.

Reply:
Fixed.